Makefile

```diff
@@ -58,6 +64,7 @@ verify: .hack_verify go_verify
 DOCKER_BUILD_TARGETS = $(addprefix docker_build_, $(CMDS))
 $(DOCKER_BUILD_TARGETS):
 	$(eval DOCKER_BUILD_CMD := $(subst docker_build_,,$@))
+	eval $$(minikube docker-env --profile $$HOSTNAME --shell sh); \
```

Comment:
❓ Is it necessary to specify a profile here? The default is minikube, which should work on a new e2e VM right?

It means that I have to remember to use $HOSTNAME as the profile, when I launch minikube locally.